avionics/common/include/rocket.h

```diff
@@ -44,6 +47,10 @@ class CommandReceiver {
   public:
     CommandReceiver(Rocket &rocket) : rocket_(rocket) {}
 
+    /**
+     * @brief sends various information from sensors based on command 
+     * @param command selects which information is sent
```

Comment:
Does not necessarily need to send info, although this is usually what happens. Can just perform some action based on a command received by the radio